saleor/graphql/core/enums.py

```diff
@@ -85,7 +85,7 @@ def to_enum(enum_cls, *, type_name=None, **options) -> graphene.Enum:
 JobStatusEnum = to_enum(JobStatus)
 PermissionEnum = graphene.Enum("PermissionEnum", get_permissions_enum_list())
 WeightUnitsEnum = graphene.Enum(
```

Comment:
Why not `JobStatus`, and `WeightUnits` as variable names?

Reply:
The goal of this task was to change enum names to keep consistency in our schema. To change variable name into `JobStatus` I also need to change imports in other places, so I would like to keep this variables name as it is.